docs/build-on-linea/linea-sdk/index.md

```diff
@@ -27,8 +27,45 @@ The SDK focuses on interacting with smart contracts on both Ethereum and Linea n
 
  5. Claiming messages (use one of the get message methods to grab all the parameters values)
 
-
-The code example below demonstrates how to initialize the SDK with the relevant configuration options, and how to call the featured methods.
+ 
+:::info[New Features]
+
+ The updated Linea SDK package enhances the L1 contract interaction and adds support for the new L1 claiming system, which is based on a Merkle tree and requires a Merkle proof for claiming.
+ 
+ There are three important things to note:
+  - The previous L1 claiming and all functions associated are still supported.
+  - The L2 claiming remains unaltered, and all SDK features to interact with L2 will remain unchanged.
+  - The previous L1 claiming function and code samples provided here cater to the transition period where pre-transition messages are claimed without the Merkle proof and post-transition with proof. If this SDK is being used after the transition, using the logic that switches between Merkle and non-Merkle proof claiming is sub-optimal.
```

Comment:
I think this whole bullet needs some work for clarity. It's quite hard to follow. I don't have any concrete suggestions as I'm not entirely sure what it's trying to say, but I think the use of "pre-transition" and "post-transition" is potentially adding complexity. Let me know if you want me to take a second look and devote more time to this.